Plugin/CppApi/source/ArcGISArView.cpp

```diff
@@ -50,13 +51,13 @@ ArcGISArView::ArcGISArView(QQuickItem* parent):
 }
 
 /*!
-  \brief A constructor that accepts an optional \a parent.
+  \internal
```

Comment:
This class can be created in C++ and the users can call this constructor. Why to set it to `\internal`?